Frameworks/CoreText/CTFont.mm

```diff
@@ -902,4 +902,31 @@ CFDataRef CTFontCopyTable(CTFontRef font, CTFontTableTag table, CTFontTableOptio
 CFTypeID CTFontGetTypeID() {
     static CFTypeID __kCTFontTypeID = _CFRuntimeRegisterClass(&__CTFontClass);
     return __kCTFontTypeID;
+}
```

Comment:
since we are adding these private apis for the sake of passing them along to xaml, we should make just one function that gets all this information.